Create community-update-november-2024.md

Grammaresque opened the pull request:
press release about 9.5 release in progress, our RISC-V news, plus docs and community news.

ressonix commented:
We'll need to figure out some of the spacing for the social icons before we can merge. Specifically, the Twitter icon. If you wanna give a shot at improving this, go for it.

ressonix commented:
I did some improvements to the language icon, and I can work on improving the social icons, but haven't yet looked into the social icons. I'll look into them shortly if you don't figure out something before then.

ressonix commented:
I'm just going to go ahead and merge this into `develop` and work on the icon improvements here shortly.

Grammaresque committed:
Create community-update-november-2024.md

press release about 9.5 release in progress, our RISC-V news, plus docs and community news.

Grammaresque committed:
Update community-update-november-2024.md

add words "RESF Partner" to description for RISC-V International.
